Fix separation of host access_ip in kubeadm SANs generation

AstromechZA opened the pull request:
When host `access_ip` is provided, we saw behavior that led to invalid Kubernetes certs due to missing separation between ip addresses due to Jinja2 directives swallowing the whitespace.

Before the change (using dummy values):

```
      kubernetes
      kubernetes.default
      kubernetes.default.svc
      kubernetes.default.svc.dns
      blerp
      localhost
      127.0.0.1
      HostA HostB
      AccessIPA
      IPAAccessIPB
      IPB
```

After the change (using dummy values):

```
      kubernetes
      kubernetes.default
      kubernetes.default.svc
      kubernetes.default.svc.dns
      blerp
      localhost
      127.0.0.1
      HostA HostB
      AccessIPA
      IPA
      AccessIPB
      IPB
```

riverzhang commented:
ci check this

riverzhang commented:
rebase code. @AstromechZA

riverzhang commented:
/lgtm

riverzhang commented:
- kubernetes
  - kubernetes.default
  - kubernetes.default.svc
  - kubernetes.default.svc.cluster.local
  - 10.233.0.1
  - localhost
  - 127.0.0.1
  - node1
  -
  - 10.58.5.131
why?   when i run it.

AstromechZA commented:
Thanks for the comments @riverzhang . I've rebased on top of origin/master now. I've also simplified the change a bit so that its doing less guesswork with the whitespace stripping.

For your manual test, what configuration where you using?

riverzhang commented:
# ## Configure 'ip' variable to bind kubernetes services on a
# ## different ip than the default iface
node1 ansible_host=147.75.42.157 ip=10.42.6.129
# ## configure a bastion host if your nodes are not directly reachable
# bastion ansible_host=x.x.x.x ansible_user=some_user

[kube-master]
node1

[etcd]
node1

[kube-node]
node1

[k8s-cluster:children]
kube-master
kube-node
